ansible/playbooks/roles/upgrade/tasks/opensearch/migrate-kibana.yml

```diff
@@ -0,0 +1,98 @@
+---
+- name: Kibana migration | Load deafults from Opensearch Dashboards role
+  include_vars:
+    file: roles/opensearch_dashboards/defaults/main.yml
+
+- name: Kibana migration | Load vars from Opensearch Dashboards role  # requires epicli upgrade -f <manifest>.yml
+  include_vars:
+    file: roles/opensearch_dashboards/vars/main.yml
+
+- name: Kibana migration | Stop Kibana service
+  systemd:
+    name: kibana
+    enabled: no
+    state: stopped
+
+- name: Kibana migration | Download Opensearch Dashboards binary
+  include_role:
+    name: download
+    tasks_from: download_file
+  vars:
+    file_name: "{{ file_name_version.opensearch_dashboards[ansible_architecture] }}"
+
+- name: Kibana migration | Create opensearch-dashboards user
+  user:
+    name: "{{ specification.dashboards_os_user }}"
+    password: "{{ specification.dashboards_password }}"
```

Comment:
Maybe it's good to follow the pattern for username here? So to specify that this is os user password?

Reply:
ok
Implemented now.